hugo/content/blog/2024-03-03-mumble-1-5-613.md

```diff
@@ -25,51 +25,38 @@ This is the changelog for this release (changes relative to the previous RC Mumb
 
 - Added: Ability to record MP3s ({{< issue 6219 >}})
 - Added: Ability to toggle audio cue exclusively for certain transmit modes ({{< issue 6005 >}})
-- Added: Increase max positional audio thresholds ({{< issue 6134 >}})
+- Added: Increase maximum positional audio distance thresholds ({{< issue 6134 >}})
 - Added: New Global Shortcuts ({{< issue 6260 >}})
 - Added: Popup when mute cue is activated for the first time ({{< issue 6155 >}})
 - Added: Setting to always show users in Talking UI ({{< issue 6158 >}})
+- Changed: Allow minimum volume of 0 for positional audio ({{< issue 6134 >}})
 - Changed: Exclude and discourage RNNoise ({{< issue 6292 >}})
-- Changed: Set minimum volume for positional audio to 0 ({{< issue 6134 >}})
+- Fixed: Allow using the keyboad to change local volume adjustment ({{< issue 6238 >}})
 - Fixed: Chinese characters not displayed properly ({{< issue 6200 >}})
 - Fixed: Code blocks end on three backticks without interruption by any other pattern of backticks in the code ({{< issue 6261 >}})
-- Fixed: Crash when opening connect dialog ({{< issue 6307 >}})
-- Fixed: Invalid code-block regex ({{< issue 6298 >}})
 - Fixed: Log incorrectly scrolling up ({{< issue 6290 >}})
 - Fixed: Logging of plugin loading errors ({{< issue 6123 >}})
-- Fixed: Migration of GKey shortcuts ({{< issue 5960 >}})
```

Comment:
This I would keep - or was this a fix for a regression caused by an unreleased commit?

Reply:
Ah. No, I just saw this in the compiler flags merge commit and assumed it was related to compiler warnings